extensions/objects/templates/svg-filename-editor.component.spec.ts

```diff
@@ -241,16 +250,12 @@ describe('SvgFilenameEditor', function() {
     expect(svgFilenameCtrl.currentDiagramWidth).toBe(MAX_DIAGRAM_WIDTH);
     svgFilenameCtrl.onHeightInputBlur();
     expect(svgFilenameCtrl.currentDiagramHeight).toBe(MAX_DIAGRAM_HEIGHT);
-  });
-
-  it('should return information on diagram size', function() {
-    var maxDiagramWidth = 491;
-    var maxDiagramHeight = 551;
-    var helpText = (
-      'This diagram has a maximum dimension of ' +
-      maxDiagramWidth + 'px X ' + maxDiagramHeight +
-      'px to ensure that it fits in the card.');
-    expect(svgFilenameCtrl.getDiagramSizeInfo()).toBe(helpText);
+    svgFilenameCtrl.diagramWidth = 0;
+    svgFilenameCtrl.diagramHeight = 0;
+    svgFilenameCtrl.onWidthInputBlur();
+    expect(svgFilenameCtrl.currentDiagramWidth).toBe(MIN_DIAGRAM_WIDTH);
+    svgFilenameCtrl.onHeightInputBlur();
+    expect(svgFilenameCtrl.currentDiagramHeight).toBe(MIN_DIAGRAM_HEIGHT);
```

Comment:
Maybe this deserves it's own "it" block? We're testing if the size gets reset to the allowed minimum if the user tries to go below a particular value and not just "updating diagram size".

Reply:
Done.